.github/workflows/tag-release.yaml

```diff
@@ -3,14 +3,38 @@ name: tag-release
 on:
   push:
     tags:
-      - "v*"
+      - "v*.*.*"
 
 jobs:
   tagged-release:
     name: Tagged Release
     runs-on: "ubuntu-latest"
 
     steps:
+      - name: Login to GitHub Container Registry
+        if: ${{ !env.ACT }}
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ secrets.CR_USER }}
+          password: ${{ secrets.CR_PAT }}
```

Comment:
This should not be needed since we're only pushing to GCR (and GAR) , we stopped producing for rudder any ghcr images since 3 months now.